src/components/map-container.js

```diff
@@ -337,11 +339,22 @@ export default function MapContainerFactory(MapPopover, MapControl) {
           .reduce(this._renderLayer, []);
       }
 
+      const tiledSampleSSDataLayer = deckGlLayers.filter(deckGlLayer => {
```

Comment:
There is a major piece missing here - eventually you want to sort and render layers by data source, not `isTiled`. Instead of making a throw-away implementation here, I encourage you to think about what is needed as a base Layer class API to support the generic use case.

Add a TODO comment that explains any assumption/missing parts here.